### DISCUSSION TITLE
Lab Assignment #3 Submission

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#123] [Assignments]
Please use this thread to post your individual `lab assignment #3`, which is the GitHub pages URL.

For example:
https://albertkun.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Comment by KenNL42** [comment]
https://kennl42.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Comment by lvalles10** [comment]
https://lvalles10.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Comment by avaitt** [comment]
https://avaitt.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Comment by rachan2023** [comment]
https://rachan2023.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Reply by alizezila** [reply]
omg the patrick stars are awesome !

**Reply by rachan2023** [reply]
thank you!! I love the zoom in effect on your website and I also think fondly of Alamo Square park!!

**Comment by alizezila** [comment]
https://alizezila.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Comment by ninalong** [comment]
https://ninalong.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Comment by rh5140** [comment]
https://rh5140.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Reply by ninalong** [reply]
this is so cute omg

**Reply by skarakas1** [reply]
Tawny frogmouth!!!

**Comment by ghost** [comment]
https://ktgreenler.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Reply by alizezila** [reply]
lol bunche is cute inside!

**Reply by ghost** [reply]
inside yes! outside.. not so much lol

**Comment by okcirE** [comment]
https://okcire.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Comment by latishasumardy** [comment]
https://latishasumardy.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Comment by vcta99** [comment]
https://vcta99.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Reply by alizezila** [reply]
this one is so sweet 😭 are all those photos taken on film? such a great map!

**Comment by nnhimichelle** [comment]
https://nnhimichelle.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Reply by nnhimichelle** [reply]
sklfksdkf i re did most of this and uploaded all my images onto wikimedia just to have these images pop up on my map markers and now it's time for me to sleep </3 if you see this good luck on midterms! and don't be like me where you're staying up and not studying for it HAHA also i've had too much coffee

**Reply by nnhimichelle** [reply]
so sorry if you see this

**Reply by albertkun** [reply]
@nnhimichelle 👀 i'm totally not awake right now working on tomorrow's lab!

**Comment by jason-muljadi** [comment]
https://jason-muljadi.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Comment by epham36** [comment]
https://epham36.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Comment by annyrviloria** [comment]
https://annyrviloria.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Reply by annyrviloria** [reply]
I have no idea why this looks so nice on my desktop but so HORRIBLE on the web. Can anyone let me know if the text appears super large?
![web](https://user-images.githubusercontent.com/72164641/115778840-15a71900-a36c-11eb-92be-fe5b0acc97ab.png)
![desktop](https://user-images.githubusercontent.com/72164641/115778851-18a20980-a36c-11eb-9376-867a41ce3c36.png)

**Comment by skarakas1** [comment]
https://skarakas1.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Comment by Wny-Duong** [comment]
https://wny-duong.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Comment by brillianz** [comment]
https://brillianz.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html

**Comment by jsxu315** [comment]
https://jsxu315.github.io/21S-ASIAAM-191A-Assignments/Week_03/index.html (Sorry about the lateness; I'll make sure to upload on time in the future)

**Comment by skarakas1** [comment]
https://skarakas1.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Reply by albertkun** [reply]
@skarakas1 psst Sam i think you posted in the wrong week.
